src/main/scala/com/yotpo/metorikku/output/writers/jdbc/JDBCQueryWriter.scala

```diff
@@ -0,0 +1,46 @@
+package com.yotpo.metorikku.output.writers.jdbc
+
+import java.sql.{Date, DriverManager}
+
+import com.yotpo.metorikku.configuration.outputs.JDBC
+import com.yotpo.metorikku.output.MetricOutputWriter
+import org.apache.spark.sql.DataFrame
+
+import scala.collection.mutable
+
+class JDBCQueryWriter(props: Map[String, String], config: Option[JDBC]) extends MetricOutputWriter {
```

Comment:
Why not just add this feature to the JDBC writer?

Reply:
I think there are more differences than similarities. If you believe they should be join it can be done.